.github/workflows/security-scanning.yml

```
@@ -0,0 +1,241 @@
+    # Run security scan weekly on Sundays at 2 AM UTC
+    - cron: '0 2 * * 0'
+
+jobs:
+  dependency-review:
+    name: Dependency Review
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)